starter-code/index.js

```diff
@@ -1,17 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items[this.length] = item;
+    this.items.sort((a, b) => a - b);
+    this.length++;
```

Comment:
this works, for this lab but keep in mind if you would like to add more than one item to your array and will just increment the length it will not be the correct length. 

you would need to write:
`this.length = this.items.length`